frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -123,4 +122,67 @@ describe('Test vm overview page', () => {
       });
     });
   });
+
+  describe('Test vm overview on project overview page', () => {
+    const newVMDescription = 'edited vm description';
+    beforeEach(async() => {
+      await browser.get(`${appHost}/overview/ns/${testName}`);
+      await isLoaded();
+    });
+
+    it('Check vm details on project overview', async() => {
+      expect(vmView.statusIcon(vmView.statusIcons.off).isPresent()).toBeTruthy();
+      await itemVirtualMachine.click();
+      await isLoaded();
+      await vm.action('Start');
+
+      expect(vmView.vmDetailDesc(testName, vmName).getText()).toEqual(newVMDescription);
+      expect(vmView.vmDetailOS(testName, vmName).getText()).toEqual(basicVmConfig.operatingSystem);
+      expect(vmView.statusIcon(vmView.statusIcons.running).isPresent()).toBeTruthy();
+      expect(vmView.vmDetailTemplate(testName, vmName).getText()).toEqual('openshift/rhel7-generic-small');
+      expect(vmView.vmDetailNamespace(testName, vmName).$('a').getText()).toEqual(testName);
+      expect(vmView.vmDetailBootOrder(testName, vmName).getText()).toEqual(['rootdisk', 'nic0', 'cloudinitdisk']);
+      expect(vmView.vmDetailFlavorDesc(testName, vmName).getText()).toEqual('2 CPU, 4G Memory');
+      expect(vmView.vmDetailIP(testName, vmName).getText()).toContain('10');
```

Comment:
Not sure if this has any point. It's okay we don't know the IP beforehand. It would be better to check that the IP has an expected format. I suppose there should be only one IP, so can we create a regular expression to match the IP? Use `toMatch(new RegExp('<regexp_here>'))`

Reply:
will do.

Reply:
instead of putting a regexp here, I change to `it should not be empty`.